code/modules/projectiles/guns/projectile.dm

```diff
@@ -33,6 +33,8 @@
 /obj/item/weapon/gun/projectile/isHandgun() //fffuuuuuuck non-abstract base types
 	return TRUE
 
+/obj/item/weapon/gun/projectile/changesilencermountpoint
```

Comment:
I assume this is not supposed to be here

Reply:
correct